src/yarn.lock

```diff
@@ -1189,10 +1189,10 @@ cssstyle@^1.0.0:
   dependencies:
     cssom "0.3.x"
 
-cuid@^2.1.0:
-  version "2.1.4"
-  resolved "https://registry.yarnpkg.com/cuid/-/cuid-2.1.4.tgz#e1178eb92a05ef9364b29a9942feed36d3604cdd"
```

Comment:
I don't think its necessary to push the yarn.lock file since its automatically generated. could we add it to the .gitingnore?

Reply:
I agree... it affects people with different node version when trying to install the packages... I did experience that until i removed the file.

Reply:
Yeah sure. Thanks for the oversight